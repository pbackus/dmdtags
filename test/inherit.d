class Base {}
interface I1 {}
interface I2 {}
interface I3 : I2 {}
class Derived : I3, Base, I1 {}
