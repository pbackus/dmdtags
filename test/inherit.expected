!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
Base	test/inherit.d	1;"	c
Derived	test/inherit.d	5;"	c	inherits:I3,Base,I1
I1	test/inherit.d	2;"	i
I2	test/inherit.d	3;"	i
I3	test/inherit.d	4;"	i	inherits:I2
inherit	test/inherit.d	1;"	M
